python/valis/db/queries.py

```diff
@@ -284,7 +292,7 @@ def carton_program_search(name: str,
     """
 
     if query is None:
-        query = vizdb.SDSSidStacked.select(vizdb.SDSSidStacked).distinct()
+        query = vizdb.SDSSidStacked.select(vizdb.SDSSidStacked)
```

Comment:
With the removal of the `distinct`, does this still return a single item per unique sdss_id?  Or now multiple items for a given sdss_id?

Reply:
Let me check. We may still want the distinct. I may have removed while doing tests to try to understand why the query with `LIMIT` is so much slower.